.github/workflows/test-r-scripts.yaml

```diff
@@ -23,9 +31,14 @@ jobs:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
 
     steps:
+      # When checking out the repository that triggered a workflow, `ref:`
+      # defaults to the reference or SHA for that event. Otherwise, uses the
+      # default branch --https://github.com/actions/checkout
       - name: Checkout PACTA_analysis
         uses: actions/checkout@v2
         with:
+          repository: 2DegreesInvesting/PACTA_analysis
+          token: ${{ secrets.MAURO_PAT_FOR_2DII }}
           path: PACTA_analysis
```

Comment:
PACTA_analysis is now treated as any other repo so that this workflow can be used anywhere. We lean on the feature by which the repo knows whether to use the master branch or the PR head.